app/javascript/src/Navigation/components/header/AccountSettingsMenu.jsx

```diff
@@ -1,6 +1,9 @@
-import React from 'react'
+// @flow
 
-const AccountSettingsMenu = ({accountSettingsLink, accountSettingsClass}) => (
+import * as React from 'react'
+import { ToolbarProps } from 'Navigation/components/header/types'
+
+const AccountSettingsMenu = ({accountSettingsLink, accountSettingsClass}: ToolbarProps): React.Node => (
```

Comment:
Returning type is not required here, it's inferred
```suggestion
const AccountSettingsMenu = ({accountSettingsLink, accountSettingsClass}: ToolbarProps) => (
```